vars/runBenchmarks.groovy

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env groovy
```

Comment:
There should be also a `runBenchmarks.txt` HTML text file. It is used for generating built-in Documentation in Jenkins IIRC